src/test/java/BlueTurtle/parsers/FindBugsXMLParserTest.java

```diff
@@ -56,17 +60,29 @@ public void testParseCorrectBehaviour() {
 	/**
 	 * Test whether the parser creates the right object.
 	 */
-//	@Test
-//	public void testParsingOneWarning() {
-//		FindBugsXMLParser parser = new FindBugsXMLParser();
-//
-//		FindBugsWarning expected = new FindBugsWarning(testSet2FilePath, testSet2FileName, 47,
-//				testSet2Message, testSet2Category, testSet2Priority, testSet2RuleName);
-//
-//		FindBugsWarning actual = (FindBugsWarning) parser.parseFile(testSet2).get(0);
-//
-//		assertEquals(expected, actual);
-//	}
+	@Test
+	public void testParsingOneWarning() {
+		FindBugsXMLParser parser = new FindBugsXMLParser();
+
+		FindBugsWarning expected = new FindBugsWarning(testSet3FilePath, testSet2FileName, 47,
+				testSet2Message, testSet2Category, testSet2Priority, testSet2RuleName, testSet2Classification);
+
+		FindBugsWarning actual = (FindBugsWarning) parser.parseFile(testSet2,categoryInfo).get(0);
+		
+		System.out.println(expected.getFileName() + " |  | " + actual.getFileName());
+		System.out.println(expected.getCategory() + " |  | " + actual.getCategory());
+		System.out.println(expected.getFilePath()); 
+		System.out.println(actual.getFilePath());
+		System.out.println(expected.getLine() + " |  | " + actual.getLine());
+		System.out.println(expected.getMessage() + " |  | " + actual.getMessage());
+		System.out.println(expected.getPriority() + " |  | " + actual.getPriority());
+		System.out.println(expected.getRuleName() + " |  | " + actual.getRuleName());
+		System.out.println(expected.getType() + " |  | " + actual.getType());
```

Comment:
This is for debugging purpose, when we ran into problems with the slashes, we had to use this way to visualize it better.